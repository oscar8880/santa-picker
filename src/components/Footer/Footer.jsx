import styles from './Footer.module.scss';
import Text from '../Text/Text';
import React from 'react';

const Footer = () => {
  return (
    <footer className={styles.Footer}>
      <Text appearance="p">Footer</Text>
    </footer>
  );
};

export default Footer;
