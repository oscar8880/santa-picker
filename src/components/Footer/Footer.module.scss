@import "../../scss/functions";
@import "../../scss/variables";

.Footer {
  align-items: center;
  background-color: $secondary-colour;
  color: $white;
  display: flex;
  height: 50px;
  justify-content: center;
}
