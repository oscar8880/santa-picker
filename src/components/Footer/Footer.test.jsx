import React from 'react';
import { render } from '@testing-library/react';

import Footer from './Footer';

test('matches snapshot', async () => {
  const { asFragment } = render(<Footer />);

  expect(asFragment()).toMatchSnapshot();
});
