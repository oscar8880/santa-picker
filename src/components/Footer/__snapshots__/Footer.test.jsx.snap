// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <footer
    class="Footer"
  >
    <p
      class="Text Text__p"
    >
      Footer
    </p>
  </footer>
</DocumentFragment>
`;
